src/main/java/net/mcreator/blockly/java/blocks/GetVariableBlock.java

```diff
@@ -61,25 +62,27 @@ public GetVariableBlock() {
 				if (scope.equals("global") && !master.getWorkspace().getVariableElements().stream()
 						.map(VariableElement::getName).collect(Collectors.toList()).contains(name)) {
 					master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-							"Variable get block is bound to a variable that does not exist. Remove this block!"));
+							L10N.t("blockly.errors.variables.invalid_var", L10N.t("blockly.block.get_var"),
+									L10N.t("blockly.errors.remove_block"))));
 					return;
 				} else if (master instanceof BlocklyToProcedure && scope.equals("local")
 						&& !((BlocklyToProcedure) master).getLocalVariables().stream().map(VariableElement::toString)
 						.collect(Collectors.toList()).contains(name)) { // check if local variable exists
 					master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-							"Variable get block is bound to a local variable that does not exist. Remove this block!"));

```

Comment:
Externalization here drops "local". I suggest you don't "merge" these translations too much.

Reply:
This should still be fixed